test-tsd/types.test-d.ts

```diff
@@ -94,17 +94,13 @@ expectAssignable<QueryBuilder>(
     .debug(true)
 );
 
-expectType<
-  QueryBuilder<
-    User,
-    DeferredKeySelection<User, 'id', true, {}, true, {}, never>[]
-  >
->(
+expectAssignable<QueryBuilder>(
   knexInstance
     .table<User>('users')
     .insert({ id: 10, active: true })
     .onConflict('id')
     .ignore()
+    .debug(true)
```

Comment:
why was debug added?

Reply:
it was a copy paste thing from the previous test, it's removed